src/components/Title/Title.styles.ts

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+interface TitleProps {
+  weight: 200 | 300 | 400 | 500 | 600 | 700;
+}
+export const STitle = styled.h2<TitleProps>`
```

Comment:
`STitle` sounds a bit confusing as a name. What does `S` stand for? It can be fine if some specific naming convention is followed.